### PR TITLE
Fix some cfg error

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py [hunk 1]
```diff
@@ -139,3 +138,47 @@
 # USER SHOULD NOT CHANGE ITS VALUES.
 # base_batch_size = (8 GPUs) x (6 samples per GPU)
 auto_scale_lr = dict(base_batch_size=48)
+
+tta_model = dict(
```
Comment (RangiLyu) [inline configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py:142]: shoud create an individual tta cfg like other models.

Comment (hhaAndroid) [reply configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py:142]: If you create separate files, file_client_args doesn't work very well, you need to change two places

File: configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 _base_ = './faster-rcnn_r50-caffe-dc5_ms-1x_coco.py'
-# learning policy
-lr_config = dict(step=[28, 34])
-runner = dict(type='EpochBasedRunner', max_epochs=36)
+
+_base_.param_scheduler[1].milestones = [28, 34]
+train_cfg = dict(max_epochs=36)
```
Comment (RangiLyu) [inline configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py:4]: Why the above one uses setting base value, while the below uses a dict-like cfg?

File: configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py [hunk 3]
```diff
@@ -1,15 +1,23 @@
 _base_ = 'faster-rcnn_r50-caffe_fpn_ms-1x_coco.py'
```
Comment (RangiLyu) [inline configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py:1]: replace with faster-rcnn_r50-caffe_fpn + common/ms-90k?

File: configs/wider_face/ssd300_24e_widerface.py [hunk 4]
```diff
@@ -3,18 +3,21 @@
...
+        type='LinearLR', start_factor=0.001, by_epoch=False, begin=0,
+        end=1000),
+    dict(
+        type='MultiStepLR',
+        begin=0,
+        end=max_epochs,
+        by_epoch=True,
+        milestones=[16, 20],
+        gamma=0.1)
+]
```
Comment (RangiLyu) [inline configs/wider_face/ssd300_24e_widerface.py:19]: refer to https://github.com/open-mmlab/mmdetection/pull/9734 
add clip grad

File: configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py (hunk 2 above)
Comment (hhaAndroid) [reply configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py:4]: The main thing is which one is convenient to use which one. Providing a diverse range of writing methods is also convenient for reference

File: configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py (hunk 1 above)
Comment (hhaAndroid) [reply configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py:142]: data_preprocessor is not processed well

File: configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py (hunk 3 above)
Comment (hhaAndroid) [reply configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py:1]: RandomChoiceResize parameter is not the same.

Comment (RangiLyu) [reply configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py:1]: ok

File: configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py (hunk 2 above)
Comment (ZwwWayne) [reply configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py:4]: then let's add some comments to explain it. for example, tell users there are another way to do that in comments.

**Comment by ZwwWayne** [comment]
overall, this pr lgtm,  as long as they are correct. We should add some comments about some usages.

**Review by RangiLyu — APPROVED** [review]

File: configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py (hunk 2 above)
Comment (hhaAndroid) [reply configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py:4]: OK

**Pull request merged**